tests/wpt/metadata/innerText/setter.html.ini

```diff
@@ -2,99 +2,51 @@
   [Simplest possible test]
```

Comment:
Do we have followups on file for this? Is the test bogus? The fact that `Simplest possible test` fails doesn't comfort me :).

Reply:
No need for followups now :) I forgot about window named getters.